### PR TITLE
[Pornorama extractor]  support site: http://www.pornorama.com/

### PR DESCRIPTION
**Pull request by Celthi**
#7850

### CODE REVIEW

File: youtube_dl/extractor/pornorama.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+from ..utils import (
+    clean_html,
+    ExtractorError
+)
+
+
+class PornoramaIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?pornorama\.com/video(?P<id>[0-9]+)(?:.*)'
+    _TEST = {
+        'url': 'http://www.pornorama.com/video1416287/amy_my_neighbor_fucked',
```
Comment (dstftw) [inline youtube_dl/extractor/pornorama.py]: This looks like copyrighted content. youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free). Do they have rights to distribute this? If not provide [an example URL](https://github.com/rg3/youtube-dl/blob/master/README.md#is-the-description-of-the-issue-itself-sufficient) that does not violate any copyrights.

**Comment by Hrxn** [comment]
Why do you believe that?

This is just a front-end for a video on xvideos?

**Comment by Celthi** [comment]
@Hrxn yeah, you are right. At first I thought this is the reason that the issue #7850 is remaining open,  piece of case to implement it. I do this for coding the simple one since I am a rookie, only for learning purpose.

File: youtube_dl/extractor/pornorama.py (hunk 1 above)
Comment (Celthi) [inline youtube_dl/extractor/pornorama.py]: @dstftw  I found  the open site support request #7850 and it is as @Hrxn  says. So I thought I may give it a try. The example URL is in the #7850. If you feel it looks like copyrighted content, please reject this pull request.

**Comment by Hrxn** [comment]
It's always a good idea to learn this stuff, I can completely understand and support your decision! :+1:

**Comment by Celthi** [comment]
@Hrxn Thank you.

**Pull request closed without merging**